packages/board-server/src/server/store.ts

```diff
@@ -28,10 +18,44 @@ export type OperationResult =
   | { success: true }
   | { success: false; error: string };
 
+const providers = {
+  "sqlite": new SQLiteStorageProvider(process.env['SQLITE_DB_PATH'] || 'board-server.db'),
+  "firestore": new FirestoreStorageProvider('board-server')
+}
+
+function getProvider(): "sqlite" | "firestore" {
+  const provider = process.env['STORAGE_BACKEND'];
+  
+  if (provider === "sqlite" || provider === "firestore") {
+    return provider;
+  }
+  
+  return "firestore";
+}
+
 export const getStore = () => {
-  return new Store("board-server");
+  return providers[getProvider()]
```

Comment:
It feels like we could combine these two functions:

```
export const getStore = () => {
  return providers[process.env['STORAGE_BACKEND']] ?? providers["firestore"]!;
}
```

(or even just an `if/else` block, which is basically the same thing with less indirection)

Reply:
Yes, that's a nice suggestion. I'll take a stab at it.